src/class-rest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * REST API class
+ *
+ * @package Parsely
+ * @since 3.1.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely;
+
+use WP_Post;
+
+/**
+ * Injects Parse.ly Metadata to WordPress REST API
+ *
+ * @since 3.1.0
+ */
+class Rest {
```

Comment:
```suggestion
class Rest_Post_Metadata {
```

Given the high likelihood we'll empower additional REST API functionality with this plugin, I suggest we be more specific with class naming.

Reply:
I don't necessarily agree with this change. We will for sure have more REST functionality, which will be spawned from a base `Rest` class. That `Rest` class will then instantiate `Rest_Post_Metadata` or whatever other features we want to implement.

At this point, we don't have other features, so I feel separating those classes is overengineering. When we do implement a second feature, I agree that we should move the specifics of the page/post metadata to its own separate class.